JobRecommendationService.md

```diff
@@ -0,0 +1,173 @@
+# Job Recommendation service
+# Contents
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+  - [JobData](#jobdata)
+    - [Job](#job)
+  - [UserData](#userdata)
+  - [UserPreferences](#userpreferences)
+    - [PreferredLocation](#preferredlocation)
+    - [Compensation](#compensation)
+    - [CaroteneTitle](#carotenetitle)
+    - [CaroteneVersion](#caroteneversion)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+----------------------------
+# Summary
+
+Job Recommendation Service is part of the profile recommendations, which recommends similar and related jobs to a user based on the jobs he applied and user preferences. The recommendations are combined from FAISS indexes and GBR data, then filtered by the applied jobs.
```

Comment:
In general, we usually try to avoid giving implementation details about our services in our docs. To that end, I would remove the reference to FAISS and GBR. I also don't really think we need to mention profile recommendations. I would just say:

> Given a job ID and, optionally, a visitor ID, email, a set of user preferences, the Job Recommendation Service provides recommendations of relevant and similar jobs.